lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -311,6 +311,14 @@ public function __construct(EntityManagerInterface $em)
         $this->reflectionPropertiesGetter = new ReflectionPropertiesGetter(new RuntimeReflectionService());
     }
 
+    /**
+     * @return IdentifierFlattener
+     */
+    public function getIdentifierFlattener(): IdentifierFlattener
```

Comment:
```suggestion
    public function getIdentifierFlattener() : IdentifierFlattener
```

Also please remove the method doc-block to fix the coding style issues mentioned in the travis-ci build.